smartGuitar/www/chords/chords.controller.js

```diff
@@ -424,18 +424,26 @@
 
             var scoreLine = "";
             var frets = [0, 0, 0, 0, 0, 0];
+            var note= [0,0];
+            var clicked_notes=0;
 
             for (var fretIdx = 0; fretIdx < vm.score[scoreIdx].length; fretIdx++) {
                 for (var stringIdx = 0; stringIdx < vm.score[scoreIdx][fretIdx].length; stringIdx++) {
                     if (vm.score[scoreIdx][fretIdx][vm.numberOfStrings - stringIdx - 1].show) {
                         scoreLine = scoreLine + "s" + (stringIdx + 1) + "f" + fretIdx + " ";
                         frets[stringIdx] = fretIdx + 1;
+                        note[0]=stringIdx; note[1]= fretIdx+1;
+                        clicked_notes++;
                     }
                 }
-
+            }
+            if (clicked_notes == 1){
+                var extracted_chord = "<" + GuitarPlayer.getSingleNote(note) + ">";
+            }
+            else{
+                var extracted_chord = GuitarPlayer.getChord(frets);
```

Comment:
Btw, shall I remove the scoreLine var?
I was to be used in the url, but for now it is just dead code